### ISSUE TITLE
wripli

### ISSUE THREAD

**Issue by jonathanjacob** [cgarwood/homeassistant-config#24]
Do you have a way to connect to wripli?

**Comment by cgarwood** [comment]
Been doing some work in the garage and haven't had a chance to finish this. I found a couple debug headers on the control board for the softener that spit out some JSON data that you could read over a serial port/gpio on a Pi. I had a little script running that tried to parse it. However, it wasn't super consistent when it would actually update, and it appears to still require a connection to Wripli's cloud to spit out that debug data. There's another set of data pins on the control board I want to checkout but haven't had any luck getting data off of them yet.

**Comment by jonathanjacob** [comment]
Interesting, thanks for the quick response. I tried a different approach but so far no luck. I am trying to see what the pairing process looks like with the iOS/Android app, and see if I can find a way to connect via the network. I will keep you up to date if I find anything. It seems they use a WICED IP board for WiFI connectivity.

**Comment by cgarwood** [comment]
Yeah being able to find a way to get the data locally without having to attach wires to the board would be awesome. Good luck!

**Comment by cgarwood** [comment]
Finally getting around to revisiting this... did you have any luck @jonathanjacob ?

**Comment by jonathanjacob** [comment]
> Finally getting around to revisiting this... did you have any luck @jonathanjacob ?

Nope :( no open ports or anything useful, even directly on the device wifi. It seems it only communicates outbound to AWS IOT cloud. 

How about you?

**Comment by jonathanjacob** [comment]
@cgarwood mind sharing a photo of the pins you used? I am circling back to this as well.
